libraries/AP_BattMonitor/AP_BattMonitor_MAV.cpp

```diff
@@ -0,0 +1,108 @@
+#include <AP_HAL/AP_HAL.h>
+#include <GCS_MAVLink/GCS.h>
+#include "AP_BattMonitor_MAV.h"
+
+AP_BattMonitor_MAVLink_Battery::AP_BattMonitor_MAVLink_Battery(AP_BattMonitor &mon,

```

Comment:
The word "battery" is redundant. Call this class AP_BattMonitor_MAV to match the filename. Again, look how other _MAV drivers do it.